packages/cli/src/index.ts

```diff
@@ -148,6 +155,9 @@ async function executeCommandLine() {
   if (update) {
     await saveTarget(+percentString)
   }
+  if (history) {
+    await saveHistory(+percentString, targetFile)
```

Comment:
If this works only when both `history` and `targetFile` are provided, they can be merged as only one option.

Reply:
Yes it required both, i've replaced them by a single `historyFile` parameter